docs/user-guide/src/bmo/ironic_variables.md

```diff
@@ -0,0 +1,26 @@
+#
+
+## Environmental variables
+
+The following environmental variables can be passed to configure the Ironic:
```

Comment:
'the Ironic services' ?
or just 'Ironic'

Reply:
services perhaps 👍